evap/results/templates/results_evaluation_detail.html

```diff
@@ -105,7 +105,7 @@ <h3>{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
                                 <tr class="current-row-follows"></tr>
```

Comment:
I noticed that the current result row is clickable when there are multiple, even if it is leading to the page currently shown. Is this behaviour wanted?
Also, we could remove the need of a `.current-row-follows` by caching a third variant. (It would then be "plain" `links_to_results_page=False`, "linked" `link_to_result_page=True` and "current")